opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java

```diff
@@ -78,13 +81,15 @@ public void testAsyncSever() throws Exception {
         response.close();
 
         List<MockSpan> mockSpans = mockTracer.finishedSpans();
-        Assert.assertEquals(2, mockSpans.size());
+        Assert.assertEquals(3, mockSpans.size());
```

Comment:
Could you please change `/async` endpoint to not return any value so the interceptor does not kick in.